xml/dm_registers.xml

```diff
@@ -17,7 +17,14 @@
         time.  Eg. in a multi-hart system some might temporarily be powered
         down, or a system might support hot-swapping harts.
 
-        <field name="0" bits="31:21" access="R" reset="0" />
+        <field name="0" bits="31:23" access="R" reset="0" />
```

Comment:
"reset" should not be 0, it should be "whatever it is".

Reply:
It's 0 for the 0 part of the register. For the impebreak it's "Preset" (see one line down).

Reply:
ah yep.